triage-party/overlays/moc/config.yaml

```diff
@@ -20,6 +20,7 @@ settings:
     - https://github.com/thoth-station/invectio
     - https://github.com/thoth-station/investigator
     - https://github.com/thoth-station/jupyterlab-requirements
+    - https://github.com/thoth-station/helm-charts
```

Comment:
good point, maybe check if we are missing more repos?!

Reply:
We certainly are: the list here has 50 repos, and there are 128 (public and not archived).

I will take a closer look at which ones to add... or should we add them all?

Reply:
I think not all are required... let's add the obvious and keep on reviewing

Reply:
For now I am adding a few more that have active issues.